File: Source/OregonTilth.Web/src/app/shared/models/forms/transplant-production-input-costs/transplant-production-input-cost-create-dto.ts
export class TransplantProductionInputCostCreateDto {
    WorkbookID: number;
    TransplantProductionInputID: number;
    TransplantProductionTrayTypeID: number;
    CostPerTray: number;
    Notes: string;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Source/OregonTilth.Models/DataTransferObjects/Workbook/Forms/TransplantProductionInputCost/TransplantProductionInputCostCreateDto.cs
using System.ComponentModel.DataAnnotations;
using System.Data.SqlTypes;

namespace OregonTilth.Models.DataTransferObjects
{
    public class TransplantProductionInputCostCreateDto
    {
        [Required] public int WorkbookID { get; set; }
        [Required] public int TransplantProductionInputID { get; set; }
        [Required] public int TransplantProductionTrayTypeID { get; set; }
        [Required] public decimal CostPerTray { get; set; }
        [Required] public string Notes { get; set; }
    }
}